messaging/src/test/java/org/axonframework/eventhandling/pooled/PooledStreamingEventProcessorTest.java

```diff
@@ -1258,7 +1254,7 @@ void coordinatorExtendsClaimsEarlierForBusyWorkPackages() throws Exception {
                .atMost(Duration.ofSeconds(5))
                .until(() -> testSubject.processingStatus().get(0).isCaughtUp());
         // Validate the token is stored
-        verify(tokenStore, timeout(5000)).storeToken(any(), eq(PROCESSOR_NAME), eq(0));
+        verify(tokenStore, timeout(5000).atLeast(1)).storeToken(any(), eq(PROCESSOR_NAME), eq(0));
```

Comment:
Ah, here it is. I thought you would've made this a separate commit on `master` already... I couldn't find it this morning, so I did this myself :sweat_smile: Hence, potential merge conflict incoming.

Reply:
Yes, resolved it